Wikipedia/Code/ReplyListItemCollectionViewCell.swift

```diff
@@ -0,0 +1,115 @@
+    weak var delegate: ReplyListItemCollectionViewCellDelegate?
+    private var theme: Theme?
+    
+    private var depth: UInt = 0
+    
+    override func sizeThatFits(_ size: CGSize, apply: Bool) -> CGSize {
+        
+        let semanticContentAttribute: UISemanticContentAttribute = traitCollection.layoutDirection == .rightToLeft ? .forceRightToLeft : .forceLeftToRight
+        let isRTL = semanticContentAttribute == .forceRightToLeft
+        let adjustedMargins = UIEdgeInsets(top: layoutMargins.top, left: layoutMargins.left + 5, bottom: layoutMargins.bottom, right: layoutMargins.right + 5)
+        
+        var depthIndicatorOrigin: CGPoint?
+        if depth > 0 {
+            var depthIndicatorX = isRTL ? size.width - adjustedMargins.right : adjustedMargins.left
+            
+            let depthAdjustmentMultiplier = CGFloat(12) //todo: may want to shift this higher or lower depending on screen size. Also possibly give it a max value
+            if isRTL {
+                depthIndicatorX -= (CGFloat(depth) - 1) * depthAdjustmentMultiplier
+            } else {
+                depthIndicatorX += (CGFloat(depth) - 1) * depthAdjustmentMultiplier
+            }
+            
+            depthIndicatorOrigin = CGPoint(x: depthIndicatorX, y: adjustedMargins.top)
+        }
+
+        var titleX: CGFloat
+        if isRTL {
+            titleX = adjustedMargins.left
+        } else {
+            titleX = depthIndicatorOrigin == nil ? adjustedMargins.left : depthIndicatorOrigin!.x + 10
+        }
+        
+        let titleOrigin = CGPoint(x: titleX, y: adjustedMargins.top)
+        var titleMaximumWidth: CGFloat
+        if isRTL {
+            titleMaximumWidth = depthIndicatorOrigin == nil ? size.width - adjustedMargins.right - titleOrigin.x : depthIndicatorOrigin!.x - adjustedMargins.left
+        } else {
+            titleMaximumWidth = (size.width - adjustedMargins.right) - titleOrigin.x
+        }
+        
+        let titleTextViewFrame = titleTextView.wmf_preferredFrame(at: titleOrigin, maximumWidth: titleMaximumWidth, alignedBy: semanticContentAttribute, apply: apply)
+        
+        let finalHeight = adjustedMargins.top + titleTextViewFrame.size.height + adjustedMargins.bottom
+        
+        if let depthIndicatorOrigin = depthIndicatorOrigin,
+            apply {
+            depthMarker.frame = CGRect(origin: depthIndicatorOrigin, size: CGSize(width: 2, height: titleTextViewFrame.height))
+        }
+        
+        return CGSize(width: size.width, height: finalHeight)
+    }
+    
+    func configure(title: String, depth: UInt) {
+        
+        self.depth = depth
+        depthMarker.isHidden = depth < 1
+        
+        let font = UIFont.wmf_font(.body, compatibleWithTraitCollection: traitCollection)
+        let boldFont = UIFont.wmf_font(.semiboldBody, compatibleWithTraitCollection: traitCollection)
+        
+        //todo: we seem to be losing <b> tags (possibly more). Need to look into this.
```

Comment:
Is this still happening? As a starting point for adding list item parsing I'm using...
> .byAttributingHTML(with: .caption2, matching: traitCollection)

... which seems to play nice with bold. 

![Screen Shot 2019-05-13 at 11 39 29 AM](https://user-images.githubusercontent.com/3143487/57645730-cc40d300-7573-11e9-8c53-f72060463dc4.png)

The params it passes to `wmf_attributedStringFromHTML` may help?

Reply:
Should be fixed now, I was accidentally overwriting the attributed string.